packages/twenty-server/src/modules/messaging/message-import-manager/drivers/imap/utils/extract-message-text.util.ts

```diff
@@ -21,10 +22,10 @@ export const extractTextWithoutReplyQuotations = (
       dom.window.document,
     );
 
-    const textContent = new JSDOM(cleanedHtml, { runScripts: 'outside-only' })
-      .window.document.body?.textContent;
-
-    return textContent ?? '';
+    return convert(cleanedHtml, {
```

Comment:
can we add a unit test on the case this is fixing?

Reply:
done